src/compass/s1_cslc_qa.py

```diff
@@ -279,32 +280,134 @@ def valid_pixel_percentages(self, cslc_h5py_root):
                                  pxl_qa_items)
 
 
-    def populate_rfi_dict(self, cslc_h5py_root):
+    def populate_rfi_dict(self, cslc_h5py_root, bursts):
         '''
         Place holder for populating SAFE RFI information
 
         Parameters
         ----------
         cslc_h5py_root: h5py.File
             Root of CSLC HDF5
+        bursts: list[Sentinel1BurstSlc]
+            List of burst SLC object with RFI info
         '''
-        rfi_qa_items = [
-            Meta('is_rfi_info_available',True,
-                 'Whether or not RFI information is available'),
-            # Follow key/values only assigned if RFI info is avaiable
-            Meta('rfi_mitigation_performed', True,
-                 'Whether or not the RFI mitigation step was performed'),
-            Meta('rfi_mitigation_domain', '',
-                 'Domain the RFI mitigation step was performed'),
-            Meta('rfi_burst_report', '', 'Burst RFI report')
-        ]
-
-        # create HDF5 group for RFI info
-        h5_rfi_path = f'{QA_PATH}/rfi_information'
-        rfi_group = cslc_h5py_root.require_group(h5_rfi_path)
 
-        # write items to HDF5 and dict
-        _qa_items_to_h5_and_dict(rfi_group, self.rfi_dict, rfi_qa_items)
+        for burst in bursts:
+            is_rfi_info_available = burst.burst_rfi_info is not None
+            rfi_qa_items_pol = [Meta('is_rfi_info_available',
+                                is_rfi_info_available,
+                                'Whether or not RFI information is available')]
+
+            if is_rfi_info_available:
+                # Follow key/values only assigned if RFI info is avaiable
+                rfi_info_list = [Meta('rfi_mitigation_performed',
+                                      burst.burst_rfi_info.rfi_mitigation_performed,
+                                      ('Activation strategy of RFI mitigation'
+                                      '["never", "BasedOnNoiseMeas","always"]')),
+                                 Meta('rfi_mitigation_domain',
+                                      burst.burst_rfi_info.rfi_mitigation_domain,
+                                      'Domain the RFI mitigation step was performed')]
+                rfi_qa_items_pol += rfi_info_list
+
+            # create HDF5 group for RFI info
+            h5_rfi_path = f'{QA_PATH}/rfi_information/{burst.polarization}'
+            rfi_group = cslc_h5py_root.require_group(h5_rfi_path)
+
+            # write items to HDF5 and dict
+            _qa_items_to_h5_and_dict(rfi_group, self.rfi_dict, rfi_qa_items_pol)
+
+            # Take care of the burst RFI report information
+            if not is_rfi_info_available:
+                return
```

Comment:
`rfi_mitigation_performed` and `rfi_mitigation_domain` are available if `is_rfi_info_available` is `False`?

Reply:
No. `rfi_mitigation_performed` and `rfi_mitigation_domain` are available only when RFI annotation exists.
If the RFI annotation do not exist, then `is_rfi_info_available` is `False`